### ISSUE TITLE
Plotting a line with x and y as list

### ISSUE THREAD

**Issue by Smit-create** [sympy/sympy#19080]
Currently, I didn't find any function for plotting a line with the data points as a list i.e `xlist` and `ylist`. Looking into the `plotting` module, I see that there is a class already implemented in the `plot.py`, but is used nowhere in the file.
https://github.com/sympy/sympy/blob/116722f31a57dac8c7e9f30c6b46c034326958bd/sympy/plotting/plot.py#L555-L556

Probably this can be used as:
```diff
diff --git a/sympy/plotting/plot.py b/sympy/plotting/plot.py
index f62cba1a3d..770002c337 100644
--- a/sympy/plotting/plot.py
+++ b/sympy/plotting/plot.py
@@ -555,12 +555,12 @@ def get_color_array(self):
 class List2DSeries(Line2DBaseSeries):
     """Representation for a line consisting of list of points."""

-    def __init__(self, list_x, list_y):
+    def __init__(self, list_x, list_y, **kwargs):
         np = import_module('numpy')
         super(List2DSeries, self).__init__()

         self.list_x = np.array(list_x)
         self.list_y = np.array(list_y)
-        self.label = 'list'
+        self.label = kwargs.get('label', None) or 'list'

     def __str__(self):
         return 'list plot'
@@ -2258,3 +2258,12 @@ def check_arguments(args, expr_len, nb_of_free_symbols):
                     raise ValueError("The ranges should be a tuple of "
                                      "length 3, got %s" % str(arg[i + expr_len]))
         return args

+
+
+def plot_list(x_list, y_list, **kwargs):
+    show = kwargs.pop('show', True)
+    series = List2DSeries(sympify(x_list), sympify(y_list), **kwargs)
+    plots = Plot(series, **kwargs)
+    if show:
+        plots.show()
+    return plots
```
possbily will be able to use as
![Screenshot from 2020-04-06 17-31-15](https://user-images.githubusercontent.com/55887635/78556993-9f517a80-782d-11ea-81e3-ae1e0257da34.png)

**Comment by sylee957** [comment]
Actually, for plotting the explicit points, you don't even need sympy and can call the matplotlib plotting directly.
```
from matplotlib.pyplot import plot
plot([1, 2, 3], [3, 4, 5])
```

**Comment by Smit-create** [comment]
In that can we remove  `class List2DSeries(Line2DBaseSeries):` as its purpose is fulfilled by `matplotlib.pyplot`?

**Comment by sylee957** [comment]
I think that plotting module is using some elaborated wrapper classes. If that is the case, sympy can have some stuff like `plot_list` that works better with sympy plotting wrappers than matplotlib even if the plot is purely numeric. But I'm afraid that things can go other directions like removing the sympy plotting wrapper classes if they are not necessary. So addition of new API should be carefully decided.

**Comment by oscarbenjamin** [comment]
I think it would be much better if sympy's plotting was a very light wrapper around matplotlib. Basically it should be a couple of convenience functions and the documentation should make clear how to e.g. get the figure/axes handles and use matplotlib directly.

If we can make it clearer to users what the relationship is between the sympy plot function and the matplotlib API then there is no need to duplicate the matplotlib API in sympy.

**Comment by Smit-create** [comment]
I agree, So, IMO we could remove those classes/functions which can be done directly using `matplotlib` to make the plotting more cleaner.
